### ISSUE TITLE
Set-HPOVEulaStatus fails  with term 'Write-Verbose9' is not recognized 

### ISSUE THREAD

**Issue by drgw0579** [HewlettPackard/POSH-HPOneView#316]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
When our code calls Set-HPOVEulaStatus, get a failure:

```PowerShell
 The term 'Write-Verbose9' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
2018-02-21T08:36:42 : ERROR: The term 'Write-Verbose9' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

The error is in HPOneView.400.psm1 line 11833:
```POwerShell

Finally
		{

			if ((${Global:ConnectedSessions} | ? Name -eq $Appliance.Name).SessionID -eq 'TemporaryConnection')
			{

				"[{0}] Removing temporary Session object" -f $MyInvocation.InvocationName.ToString().ToUpper(), $Caller | **Write-Verbose9**

				[void]${Global:ConnectedSessions}.Remove($Appliance)

			}

		}
```

### Steps to reproduce

```PowerShell
# 1. Capture verbose output using the Get-HPOVCommandTrace for HPOneView.310 or newer, or append the -Verbose switch to your Cmdlet call.
# 2. Put your code and/or captured output here.
# 3. Remove this comment block before submitting.
```

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion` or `$PSLibraryVersion`): 4.0.1587.3400
HPE OneView Appliance Version (`Get-HPOVVersion -ApplianceVer`):   
Output from `$PSVersionTable` on your Windows Host:

```PowerShell
# Output from $PSVersionTable
```

**Comment by ChrisLynchHPE** [comment]
Thanks.  I'll have this fixed in the next release.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 4.0.1612.2800](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v4.0.1612.2800) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.400/4.0.1612.2800).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing this issue due to no further input.  If this is still an issue for you, we will re-open to look at further.
